inst/htmlwidgets/Leaflet.glify/addGlifyPolygons.js

```diff
@@ -9,6 +9,7 @@ LeafletWidget.methods.addGlifyPolygons = function(data, cols, popup, opacity, gr
       clrs = function(index, feature) { return cols[index]; };
     }
 
+/*
     var pop;
     if (popup) {
```

Comment:
Why did you comment this out? this means that we can use `popup = TRUE` as a short-hand to show all feature entries in the popup without having to create it on the R side first. Is there a reason fro not allowing this?

Reply:
`popup = TRUE` is still possible, but runs now through the popup-method and creates a somewhat similar popup-table. Like that the JS-functions look more alike, which makes it easier to debug and implement new features I think.

Reply:
The reason for the specific JS popup method not working for points is that currently the upstream method does not support geojson point format, only flat coordinate arrays. I hope that this will be resolved in the future. There is already an open issue regarding this point. Maybe @robertleeplummerjr has more thoughts on whether he plans to support geojson point objects in the future. 

My thinking here is that popup creation on the R side may slow things down quite significantly for large objects, that's why I have my reservations about this...

Reply:
I could do some benchmarks with the current version and the popup-method, to see the differences. The benefit of the popup-method is that the popup is completely customizable.

Reply:
We can still use both I think. Use the JS method only if `popup === true` else use the `popup[idx].toString()` method. This is basically what the whole `var pop` definition is all about.

Reply:
But benchmarks would be good in any case!

Reply:
First benchmarks indeed dont look too good. The popup-method is about 10x slower than the JS-version. But maybe the `apply(x, 1, function)` can still be optimized. I didnt really focus on performance to be honest.

Reply:
Why not use `leafpop::popupTable` in case you really want to do it in R?

Reply:
No, I am fine with the idea of using the JS method when popup is TRUE. I personally, would mostly use a formula for the popups, as I need to define a certain title, or wrap the content in css-classes etc.